### PR TITLE
fix the Dockerfile deprecation issue

### PR DESCRIPTION
**Pull request by dminca**
The `MAINTAINER` option was deprecated long time ago in favor of `LABEL`

Resolves:
Related:

### CODE REVIEW

**Comment by dminca** [comment]
can't sign the CLA, don't have any instruction on that lol

**Comment by delgod** [comment]
@dminca could you try https://cla-assistant.io/percona/percona-xtradb-cluster-operator?pullRequest=185

**Comment by dminca** [comment]
> @dminca could you try https://cla-assistant.io/percona/percona-xtradb-cluster-operator?pullRequest=185

Thank you, that worked 👍

File: build/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 FROM registry.access.redhat.com/ubi7/ubi-minimal
 RUN microdnf update && microdnf clean all
 
-MAINTAINER Percona Development <info@percona.com>
 LABEL name="Percona XtraDB Cluster Operator" \
       vendor="Percona" \
       summary="Percona XtraDB Cluster is an active/active high availability and high scalability open source solution for MySQL clustering" \
-      description="Percona XtraDB Cluster is a high availability solution that helps enterprises avoid downtime and outages and meet expected customer experience."
+      description="Percona XtraDB Cluster is a high availability solution that helps enterprises avoid downtime and outages and meet expected customer experience." \
+      MAINTAINER="Percona Development <info@percona.com>"
```
Comment (delgod) [inline build/Dockerfile]: lowercase according to the example in the official doc - https://docs.docker.com/engine/reference/builder/
```suggestion
      maintainer="Percona Development <info@percona.com>"
```

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=185) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=185) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-xtradb-cluster-operator?pullRequest=185) it.</sub>

**Pull request merged**